views/personal/maritalStatus-edit.pug

```diff
@@ -10,27 +10,18 @@ block content
   div
     p Please choose the correct status from the options listed below.
 
+  //- this is messy (data && data.personal && data.personal.maritalStatus) but pug will error if a property doesn't exist. We could handle this on the controller level, and pass this to the route? Will help keep it a bit cleaner here
+  - var radioFormOther = {common: { name: 'maritalStatus', dataKey: (data && data.personal && data.personal.maritalStatus ? data.personal.maritalStatus : null) }, options: maritalStatusOptions.en }
```

Comment:
Is `dataKey` the same as the "value" html attribute? (ie, could we call it `value`)

Reply:
so datakey is checking the data as we enter the page it's basically used to run the check:
`checked=(radioForm.common.dataKey &&radioForm.common.dataKey.toLowerCase()==option.value.toLowerCase()))` to see if the item should be checked when we load the page (ex. someone updating their marital status where previously it was 'Single' and so when they get to the page 'Single' will be checked before they change it).

We could get rid of pre-checking altogether, or handle it differently! I'm open to alternatives as making sure that first `data` exists, and then `data.personal` exists, and then `data.personal.maritalStatus` exists. We could offload this series of checks into a function that's in utils?